### PR TITLE
feat(fetch as): Added fetch and open function on monitoring and code refactoring on send to pane

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
SD-3230

HTML and CSS Refactoring was done by [BEM](https://css-tricks.com/bem-101/) convention.

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/views/send-item.html [hunk 1]
```diff
@@ -1,77 +1,98 @@
...
+<div class="slide-pane slide-pane--dark" ng-show="isActive" ng-class="{open: isActive, 'slide-pane--inset': mode === 'authoring'}">
+
+    <div class="slide-pane__header">
+        <h3 ng-show="mode === 'authoring'" translate>Send to / Publish</h3>
+        <h3 ng-show="mode !== 'authoring'" translate>Fetch To</h3>
+        <div class="close" ng-click="close()"><i class="icon-close-small icon-white"></i></div>
+    </div>
+
+    <div class="slide-pane__content">
+        <div sd-toggle-box ng-if="showSendButtonAndDestination()" data-title="{{ 'Destination' | translate }}" data-open="true" data-style="dark">
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/send-item.html]: One time binding for the fixed title string is missing here.

**Comment by pavlovicnemanja** [comment]
ping @superdesk/developers

**Comment by eozan** [comment]
:+1:

**Pull request merged**